conda/recipes/cudf/meta.yaml

```diff
@@ -25,20 +25,20 @@ requirements:
   #   - defaults
   build:
     - python
-    - cython=0.28.*
+    - cython>=0.29
     - setuptools
     - libcudf 0.4.0.*
     - libcudf_cffi 0.4.0.*
-    - numba>=0.40.0dev
+    - numba>=0.40
```

Comment:
For some reason I thought we started using 0.41 recently, just wanted confirmation that 0.40 is fine.

Reply:
0.40.0 release is fine while 0.40.0dev has some bugs. Ideally this should allow 0.40.0, 0.41.0, 0.42.0, etc. without grabbing dev or rc versions.